pybedlite/overlap_detector.py

```diff
@@ -100,6 +102,49 @@ def length(self) -> int:
         """Returns the length of the interval."""
         return self.end - self.start
 
+    def to_bedrecord(self) -> BedRecord:
```

Comment:
Would there be objections to `to_bed` and `from_bed` instead of `to_bedrecord` and `from_bedrecord` ?

Reply:
No objections, since we have a type to help clarify